### PR TITLE
BA-1516-010: Representation on CORe

### PR DESCRIPTION
**Pull request by DakotaNelson**
These are easier to comment/iterate on when they're pull requests.

### CODE REVIEW

File: student_government_bylaws.md [hunk 1]
```diff
@@ -87,156 +87,137 @@ Council and the Student Government. The executive officers are:
...
-accessible to the Student Body, as specified in [Article
-13](https://github.com/olin/studentgovernment/blob/master/student_government_bylaws.md#article-13-transparency).
-The Vice President for Communications will advertise Student Government and
-Council related events, organize the publishing of the monthly report, and
-actively encourage Student Body participation in the improvement of the Student
-Government and Olin culture.
-
-**Vice President for Finance.** The Vice President for Finance will keep the
+the Student Government and the Council. The President moderates Town
+Meetings called by the Student Government. Along with the Vice President for
```
Comment (mathmitch7) [inline student_government_bylaws.md]: This seems like a super random addition. New accounts are created whenever somebody donates money to the college for a particular purpose, so we don't necessarily have control over this. What was the motivation for this change?

**Comment by mathmitch7** [comment]
I remember at some point the conversation around reforming to mostly specialized reps included some safeguard to ensure that every class would still be represented. Was such a safeguard forgotten, or deemed unnecessary, or something else?

**Comment by LivKelley** [comment]
Hi Mitch,

     Such a provision should have been included. I'll double check to make sure it got in there.

         Thanks,

                Liv

---

________________________________

**Comment by LivKelley** [comment]
Hi All,

   Here's the clause about the safeguard to make sure every class is represented.

https://github.com/olin/studentgovernment/pull/11/files#diff-0965de7b21eaacaa76d7182148cb2461R926.<https://github.com/olin/studentgovernment/pull/11/files#diff-0965de7b21eaacaa76d7182148cb2461R926>
        Thanks!
              Liv
P.S. It's in the end in the "voting" section.?
________________________________

**Comment by mathmitch7** [comment]
I'm still wondering what happens if nobody from a particular class runs for office?

**Comment by iangmhill** [comment]
Approved by the Student Government on April 21, 2016

**Pull request merged**